pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -68,7 +70,7 @@ apiVersion: console.openshift.io/v1
 auth:
   clientID: console
   clientSecretFile: /var/oauth-config/clientSecret
-  oauthEndpointCAFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+  oauthEndpointCAFile: /var/router-ca/ca-bundle.crt
```

Comment:
I think we may want to add 2 new tests here:

```
name: "Test configmap with router-ca",
name: "Test configmap without router-ca",
```
Probably can duplicate the test `name: "Test default configmap, no customization",` and just swap the bool & output.  This would make it explicit.